hooks/post-command

```diff
@@ -45,22 +58,24 @@ main() {
   fi
 
   local ci_env
-  ci_env=$(bash <(curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/env))
+  ci_env=$(bash $DIR/../scripts/env)
 
   set +e
-  local exit_code  
+  local exit_code
   docker run \
       $ci_env \
       --label "com.buildkite.job-id=${BUILDKITE_JOB_ID}" \
       --workdir=/workdir \
       --volume=$(pwd):/workdir \
+      --volume=${DIR}/codecov:/codecov \
       -it \
       --rm \
       buildpack-deps:jessie-scm \
-      bash -c "bash <(curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/bash) ${args[*]:-}"
+      bash -c "bash /codecov ${args[*]:-}"
   exit_code="$?"
   set -e
   popd >/dev/null
+  rm codecov
```

Comment:
I think we'll need to cache this on the agent.
I can't find official information for unauthenticated requests, but it seems to be between 60 and 5000 requests/hour?: https://github.community/t/raw-githubusercontent-com-rate-limit/142444

Even at 5000 that would not be high enough to fetch it every single time.

I have another plugin that does such caching already: https://github.com/joscha/sauce-connect-buildkite-plugin/blob/master/hooks/pre-command - we can probably just adapt the logic.

Reply:
Will take a look